infra/base-images/base-builder/compile_libfuzzer

```diff
@@ -25,4 +25,5 @@ $CXX $CXXFLAGS -std=c++11 -O2 -fPIC $SANITIZER_FLAGS -fno-sanitize=vptr \
 ar r $LIB_FUZZING_ENGINE $WORK/libfuzzer/*.o
 popd > /dev/null
 rm -rf $WORK/libfuzzer
+export $LIB_FUZZING_ENGINE="-fsanitize=fuzzer"
```

Comment:
Please add a comment like "# Override `LIB_FUZZING_ENGINE` variable as we use compiler instrumentation flag instead of pre-built library for libFuzzer builds". You also probably need to remove `$` from this line.